recipes/xcape/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "xcape" %}
+{% set version = "0.1" %}
+{% set sha256 = "288dd48bf9257cf7afed92c45b6c98b50db60e5cd13cd4d964e2522c6add84bd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/xgcm/xcape/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<36 or win]
```

Comment:
Do you mean to be skipping `win` here? The `build` section listing a fortran for windows seems contradictory.

Reply:
We copied this directly from https://github.com/conda-forge/xlayers-feedstock/blob/6fb15babf22e3cf81d2d0531ff86c5fea69499d9/recipe/meta.yaml#L11-L14, which was written by @ocefpaf. TBH, I'm not exactly sure what it does. Should I try removing?

Reply:
We left that in xlayers b/c we do plan to try Windows again later. It is harmless to leave it here too.